.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
```

Comment:
- [ ] [REQUIRED] Your README file is missing some other sub-heading. Please add the other headings as seen in this [sample](https://github.com/[microverseinc/readme-template](https://github.com/microverseinc/readme-template)). This is very important because it will be required of you in all your projects going forward, so we want to be sure you understand it now. Kindly edit it to suit your project.

Reply:
Added more relevant subheadings in my README